packages/tasks/src/model-libraries.ts

```diff
@@ -852,6 +852,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/revdotcom/reverb",
 		filter: false,
 	},
+	"rkllm": {
+		prettyLabel: "RKLLM",
+		repoName: "rknn-llm",
```

Comment:
```suggestion
		repoName: "RKLLM",
```

let's keep the same as `prettyLabel` (sorry about that, in practice we want to get rid of this duplicated field